docs/architecture/tablets.rst

```diff
@@ -0,0 +1,99 @@
+=========================================
+Data Distribution with Tablets
+=========================================
+
+A ScyllaDB cluster is a group of interconnected nodes - on-premises servers or 
+virtual servers (public cloud instances). The data of the entire cluster has to
+be distributed as evenly as possible across those nodes.
+
+ScyllaDB is designed to ensure a balanced distribution of data by storing data
+in tablets. When you add or remove nodes to scale your cluster, add or remove
+a datacenter, or replace a node, tablets are moved between the nodes to keep
+the same number on each node.
+
+This article explains the concept of tablets and how they let you scale your
+cluster quickly and seamlessly.
+
+Data Distribution
+-------------------
+
+ScyllaDB distributes data by splitting tables into tablets. Each tablet has 
+its replicas on different nodes, depending on the RF (replication factor). Each
+partition of a table is mapped to a single tablet in a deterministic way. When you
+query or update the data, ScyllaDB can quickly identify the tablet that stores
+the relevant partition. 
+
+The following example shows a 3-node cluster with a replication factor (RF) of
+3. The data is stored in a table (Table 1) with two rows. Both rows are mapped
+to one tablet (T1) with replicas on all three nodes.
+
+.. image:: images/tablets-cluster.png
+
+.. TODO - Add a section about tablet splitting when there are more triggers,
+   like throughput. In 6.0, tablets only split when reaching a threshold size
+   (measured in the number of rows).
+
+Load Balancing
+==================
+
+ScyllaDB autonomously moves tablets to balance the load based on table usage. 
```

Comment:
1. This is very vague.  What does "table usage" mean?
2. At the moment, tablets are migrated only to balance the number of tablets across nodes or across shards in a single node.  The only response to the workload in 6.0 is tablet splitting when the table grows, and then the split-halves might be migrated away, independently, to balance the number of tablets as per the above.